CPAN/DBD-SQLite-XOPEN.patch

```diff
@@ -0,0 +1,11 @@
+--- sqlite3.c.orig	Mi. Nov  1 21:02:52 2017
++++ sqlite3.c	Mi. Nov  1 21:05:54 2017
+@@ -425,7 +425,7 @@
+ ** See also ticket #2741.
+ */
+ #if !defined(_XOPEN_SOURCE) && !defined(__DARWIN__) && !defined(__APPLE__) && SQLITE_THREADSAFE
+-#  define _XOPEN_SOURCE 500  /* Needed to enable pthread recursive mutexes */
```

Comment:
What is this supposed to do?

Reply:
The DBD-SQLite version supplied in the repo uses an "_XOPEN_SOURCE 500" define this causes newer gcc's to refuse working as their expect "_XOPEN_SOURCE 600" in the feature test macros. DBD-SQLITE-1.54 (latest stable) defines "_XOPEN_SOURCE 600" and then compilation is okay. Settings a compiler define (-D...) caused perl to refuse working after compilation.
I thought of staying with DBD-SQLite 1.34 as this has proven to work...

Reply:
Just saw in the script that it's SunOS only (for now) anyway.